js/source/tile.js

```diff
@@ -20,11 +21,10 @@ function Tile(coord, size, sourceMaxZoom) {
     this.uses = 0;
     this.tileSize = size;
     this.sourceMaxZoom = sourceMaxZoom;
+    this.pixelRatio = EXTENT / size;
```

Comment:
It was used for part of the conversion from pixels to tile units. 37e540f cleans up how that happens and removes `tile.pixelRatio`.